### PR TITLE
Add license of toxiclibsjs, cc #5500

### PR DESCRIPTION
**Pull request by sufuf3**
https://github.com/hapticdata/toxiclibsjs/blob/master/LICENSE.txt

### CODE REVIEW

File: ajax/libs/toxiclibsjs/package.json [hunk 1]
```diff
@@ -11,5 +11,6 @@
   "repository": {
     "type": "plain file",
     "url": "https://raw.github.com/hapticdata/toxiclibsjs/master/build/toxiclibs.min.js"
-  }
+  },
+  "license": "GPLv2"
```
Comment (PeterDaveHello) [inline ajax/libs/toxiclibsjs/package.json]: That's now correct, please check it again.

**Comment by sufuf3** [comment]
@pvnr0082t  please help me review it, thanks!

**Comment by PeterDaveHello** [comment]
Still wrong

> 
>		  GNU LESSER GENERAL PUBLIC LICENSE
>		       Version 2.1, February 1999

**Comment by sufuf3** [comment]
@pvnr0082t  please help me review it, thanks!

**Pull request merged**